src/overlays/actors/ovl_En_Brob/z_en_brob.c

```diff
@@ -54,37 +61,285 @@ static ColliderCylinderInit D_809CBA80 = {
     },
     { 8000, 11000, -5000, { 0, 0, 0 } },
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Brob/EnBrob_Init.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Brob/EnBrob_Destroy.s")
+static CollisionCheckInfoInit sCylinderInit = { 0, 60, 120, MASS_IMMOVABLE };
```

Comment:
This should be named `sColChkInfoInit`. `sCylinderInit` should be the collider init.